docs/users/cli-usage.md

```diff
@@ -221,9 +221,24 @@ Docroot Location: web
 Found a drupal8 codebase at /Users/username/Projects/drupal8/web
 ```
 
-Configuration files have now been created for your project. (Take a look at the file on the project's .ddev/ddev.yaml file).
+Configuration files have now been created for your project. (Take a look at the file on the project's .ddev/ddev.yaml file). Additionally, the `ddev config` steps attempts to create a CMS specific database settings with the DDEV specific credentials pre-populated. Here's a Drupal specific example that is mirrored in the other CMSes.
 
-Now that the configuration has been created, you can start your project with `ddev start` (still from within the project working directory):
+* If settings.php does not exist, create it.
+* If a settings.php file exists that DDEV manages, recreate it.
+* If a settings.php file exists that DDEV does not manage, create settings.local.php.
+* If a settings.local.php file exists that DDEV manages, recreate it.
+* If a settings.local.php file exists that DDEV does not manage, warn the user and proceed.
+
+How do you know if DDEV manages a database settings file? You will see the following comment. Remove the comment and DDEV will not attempt to overwrite it!
+
+```
+/**
+ #ddev-generated: Automatically generated Drupal settings.php file.
+ ddev manages this file and may delete or overwrite the file unless this comment is removed.
+ */
+```
+
+Now that the configuration files have been created, you can start your project with `ddev start` (still from within the project working directory):
```

Comment:
Since we're on the subject and have never mentioned this in docs before, why don't we say that the whole strategy fails if settings.php doesn't have the include of settings.local.php. Same basic idea on Wordpress and Backdrop. Not relevant to TYPO3.

Reply:
Good point and this might be even more complicated. First, backdrop already has the local include by default https://github.com/backdrop/backdrop/blob/1.9.x/settings.php. WordPress has some issues to work out in https://github.com/drud/ddev/issues/742. TYPO3 is all good.

Let's scope this to include the CMS specific call outs and perhaps reference issues that exist and that need to get updated/removed when resolved (WordPress).

I'll take another stab at this after planning meeting.